policies/privacy-policy.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Privacy Policy
+layout: layouts/policy.njk
+---
+
+Version 1.2, dated 2021-04-16
+
+The Rust Foundation maintains several services related to the Rust Project, including the[ rust-lang.org](https://www.rust-lang.org/),[ crates.io](https://crates.io), and[ docs.rs](https://docs.rs) (the “Services”). This privacy notice explains what we do with personal information.
```

Comment:
```suggestion
The Rust Foundation maintains several services related to the Rust Project, including the [rust-lang.org](https://www.rust-lang.org/), [crates.io](https://crates.io), and [docs.rs](https://docs.rs) (the “Services”). This privacy notice explains what we do with personal information.
```

This seems to be a common thread for multiple links fwiw, and it affects rendering - but the rendering of underlining the space is not horrible so feel free to leave as-is. I think it's probably not *too* hard to fix with a regex.